nixos/modules/services/monitoring/todesk.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.services.todesk;
+
+in
+
+{
+
+  ###### interface
+  options = {
+
+    services.todesk.enable = lib.mkEnableOption "ToDesk daemon";
+
+  };
+
+  ###### implementation
+
+  config = lib.mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.todesk ];
+
+    systemd.services.todeskd = {
+      description = "ToDesk Daemon Service";
+
+      wantedBy = [ "multi-user.target" ];
+      wants = [ "network-online.target" "nss-lookup.target" ];
+      after = [ "network-online.target" ];
+      before = [ "nss-lookup.target" ];
```

Comment:
why does this service need to start before "A target that should be used as synchronization point for all host/network name service lookups" (https://www.freedesktop.org/software/systemd/man/latest/systemd.special.html)